### DISCUSSION TITLE
Duda ejercicio1 tp2

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#58] [Preguntas y respuestas] [answered]
Consulta esta bien planteado lo siguiente:
https://github.com/INGCOM-UNRN-P1/tp2-2023-NicoHermosilla
Quiero saber si esta bien planteado definir el error al principio de la funcion, para atajar cualquier caso que genera que el codigo no funcione o hacer la logica en el main para que cuando el usuario ingrese el valor haga la comprobacion ahi.

**Comment by martinvilu** [comment, marked as answer]
Mejor, retorna un valor negativo en lugar de un printf

Reemplaza la imagen por un enlace al repositorio.
